v2/azure-arm.yaml

```diff
@@ -936,17 +931,20 @@ objectModelConfiguration:
       FlexibleServers_FirewallRule:
         $exportAs: FlexibleServersFirewallRule
         $supportedFrom: v2.0.0-alpha.2
-    2021-12-01-preview:
+    2022-01-01:
       FlexibleServers_Administrator:
         $exportAs: FlexibleServersAdministrator
-        $supportedFrom: v2.0.0
+        $supportedFrom: v2.1.0
```

Comment:
Looks like we released this in v2.0.0, probably need to retain it in v2.1.0

Reply:
Unless we don't get a choice because the underlying OpenAPI schema has gone.

Reply:
That was a copy-paste mistake. It's actually 2.1.0 (which is why it's safe for me to remove)

Reply:
For completeness, it was in #2954, which was only merged 5 days ago.

Reply:
That works. 👍